spyder/widgets/sourcecode/codeeditor.py

```diff
@@ -2624,6 +2632,18 @@ def __in_comment(self):
         else:
             return False
 
+    def __in_string(self):
+        """Check if we are in a string."""
+        if self.highlighter:
+            current_color = self.__get_current_color()
+            comment_color = self.highlighter.get_color_name('string')

```

Comment:
why is this called comment_color ?

Reply:
I took the lines from the `__in_comment` method and forgot to change the name of the variable. Thank you for noticing it :+1: